README.md

```diff
@@ -1,15 +1,56 @@
 Chitter Challenge
```

Comment:
I think this README is really detailed and makes it easy to navigate your project. :)